test/integration/app/prompted_build.js

```diff
@@ -96,4 +98,77 @@ describe('Prompt and build integration tests for app generator', function () {
     });
   });
 
+  describe('Starter with generated SDK integration', function () {
+    var runContext;
+    var appName = 'notes';
+    before(function () {
+      // Swift build is slow so we need to set a longer timeout for the test
+      this.timeout(buildTimeout);
+      runContext = helpers.run(appGeneratorPath)
+                          .withPrompts({
+                            appType: 'Scaffold a starter',
+                            name: appName,
+                            dir: appName,
+                            appPattern: 'Basic',
+                            endpoints: 'Endpoints from swagger file',
+                            swaggerChoice: 'Custom swagger file',
+                            path: testResourcesPath + '/petstore.yaml',
+                            serverSwaggerInput0: true,
+                            serverSwaggerInputPath0: testResourcesPath + '/petstore.yaml',
+                            serverSwaggerInput1: true,
+                            serverSwaggerInputPath1: testResourcesPath + '/petstore2.yaml',
+                            serverSwaggerInput2: false
+                          });
+      return runContext.toPromise();                        // Get a Promise back when the generator finishes
+    });
+
+    it('compiles the application', function () {
+      assert.file(process.cwd()+'/.build/debug/' + appName);
+    });
+
+    it('generates an .xcodeproj file', function () {
+      assert.file(appName + '.xcodeproj');
+    });
+
+    it('created a iOS SDK zip file', function() {
```

Comment:
Yes, there should be a compile test for sure. However, the asserts below this point do not need a compilation to test, so they should be in a nobuild test so that can be run under `npm run testintfast`.